rules/aws_cloudtrail_rules/aws_suspicious_saml_activity.yml

```diff
@@ -0,0 +1,140 @@
+AnalysisType: rule
+Description: Identifies when suspicious SAML activity has occurred in AWS. An adversary could gain backdoor access via SAML.
```

Comment:
I would replace "suspicious" and explicitly call out which activities we're catching.  I might make sense to say "Identifies when SAML IAM configuration has been created or modified"